statics/js/components/preference.js

```diff
@@ -0,0 +1,98 @@
+/* jshint multistr: true */
+
+var PreferenceComponent = {
+
+  name: 'Preference',
+
+  mixins: [notificationMixin],
+
+  template: '\
+    <form class="form-preference" @submit.prevent="save">\
+      <h2> Preferences </h2>\
```

Comment:
I would remove the title as we already know that we are editing the preferences, we clicked on the tab and the tab is selected.

Reply:
done